lib/segment/tests/integration/sparse_vector_index_search_tests.rs

```diff
@@ -170,20 +180,59 @@ fn sparse_vector_index_ram_consistent_with_storage() {
         let top = sparse_vector_index.max_result_count(vector);
         let query_vector: QueryVector = vector.to_owned().into();
         let results = sparse_vector_index
-            .search(&[&query_vector], None, top, None, &stopped)
+            .search(&[&query_vector], None, top, None, &false.into())
             .unwrap();
         assert!(results[0].iter().any(|s| s.idx == id));
     }
 }
 
+#[test]
+fn sparse_vector_index_consistent_with_storage() {
+    let stopped = AtomicBool::new(false);
+    let mut rnd = StdRng::seed_from_u64(42);
+
+    let sparse_vector_ram_index = fixture_sparse_index_ram(&mut rnd, MAX_SPARSE_DIM, &stopped);
+
+    // check consistency with underlying RAM inverted index
+    check_index_storage_consistency(&sparse_vector_ram_index);
+
+    let mmap_index_dir = Builder::new().prefix("mmap_index_dir").tempdir().unwrap();
+    // copy index mmap and save to disk
+    let _mmap_inverted_index = InvertedIndexMmap::convert_and_save(
```

Comment:
important moment here

Reply:
I guess we don't have to keep this index alive. I propose to remove `_mmap_inverted_index` variable. And next loading will be processed when this index is dropped